docs/release-notes/NuGet-6.9.md

```diff
@@ -0,0 +1,112 @@
+
+### Breaking changes
+
+* Add nullable annotations to NuGet.LibraryModel - [#12889](https://github.com/NuGet/Home/issues/12889)
+
+### Issues fixed in this release
+
+* NuGetAudit should not download vulnerabilities database when project does not use any packages - [#13073](https://github.com/NuGet/Home/issues/13073)
+
+* Static graph-based restore should not enumerate every item's metadata - [#13049](https://github.com/NuGet/Home/issues/13049)
+
+* Migrate NuGet.CommandLine.XPlat package search to use System.CommandLine - [#13031](https://github.com/NuGet/Home/issues/13031)
+
+* Set NuGetAudit defaults in MSBuild - [#12960](https://github.com/NuGet/Home/issues/12960)
+
+* RestoreTask: Control whether to embed files in the binlog - [#12957](https://github.com/NuGet/Home/issues/12957)
+
+* Create NU Error Code for Package Source Mapping & GPF conflicts - [#12953](https://github.com/NuGet/Home/issues/12953)
+
+* [DCR]: Allow floating versions with Central Package Management (CPM) - [#10432](https://github.com/NuGet/Home/issues/10432)
+
+* Static Graph restore won't warn for invalid projects - [#9300](https://github.com/NuGet/Home/issues/9300)
+
+* Rename no-cache to no-http-cache - [#9180](https://github.com/NuGet/Home/issues/9180)
+
+* VS Package Manager Console should close Text View - [#13104](https://github.com/NuGet/Home/issues/13104)
+
+* Vulnerability indicator shows up on dependent project if parent project has transitive vulnerabilities - [#13068](https://github.com/NuGet/Home/issues/13068)
+
+* Conditional updating when *all*  packages are conditioned is broken - [#13034](https://github.com/NuGet/Home/issues/13034)
+
+* Perf issue in AssetsFileDependenciesDataSource - [#13019](https://github.com/NuGet/Home/issues/13019)
+
+* The `NuGetPackageSearchService.GetDeprecationMetadataAsync` in `NuGet.PackageManagement.VisualStudio` is dead code - [#13007](https://github.com/NuGet/Home/issues/13007)
+
+* Vulnerabilities filter shows incorrectly on “Brower” tab when the default tab was “Browse” instead of “Installed” for the previous opening of solution PM UI   - [#12974](https://github.com/NuGet/Home/issues/12974)
+
+* HTTP 401 after some time in VS - [#12961](https://github.com/NuGet/Home/issues/12961)
+
+* [NuGet.Versioning] SemanticVersion.HasMetadata should indicate that Metadata is not null when true - [#12949](https://github.com/NuGet/Home/issues/12949)
+
+* TelemetryUtility.ToJsonArrayOfTimingsInSeconds returns incorrect json array on locales having comma as decimal separator - [#12915](https://github.com/NuGet/Home/issues/12915)
+
+* Static graph-based restore does not respect Interactive option when loading projects - [#12907](https://github.com/NuGet/Home/issues/12907)
+
+* Vulnerabilities InfoBar link to `Manage NuGet Packages` is truncated - [#12835](https://github.com/NuGet/Home/issues/12835)
+
+* NuGet.Build.Tasks caches CredentialProvider device flow timeout. - [#12540](https://github.com/NuGet/Home/issues/12540)
```

Comment:
There's a bunch of auth improvements, do we need a `What's new` item summarizing the changes. 
Is that noteworthy enough @kartheekp-ms

Reply:
Thanks, Nikolche for the suggestion. Unfortunately, the auth improvements that have good impact are in the next release i.e. 6.10.https://github.com/NuGet/Home/issues/12540 bug fix improved the authentication experience in an edge case and only applies to dotnet.exe, hence I am okay to not to mention about it in the "What's New" section.

1. https://github.com/NuGet/NuGet.Client/commit/0139b8931f3e8404a29016a499155978be4f98a9
2. https://github.com/NuGet/NuGet.Client/commit/c13cea4dfd50e7eba1b794e1fea979dd1d6af3bd